### PR TITLE
Enable eBPF mode 

### PR DESCRIPTION
**Pull request by ghost**
Add tabs to differentiate Operator from Manifest instructions for consistency

Product Version(s):
Calico only

Issue:
[CORE-9870](https://tigera.atlassian.net/browse/CORE-9870)

Link to docs preview:
[Enabling eBPF](https://docs.tigera.io/calico/latest/operations/ebpf/enabling-ebpf)

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed 

[CORE-9870]: https://tigera.atlassian.net/browse/CORE-9870?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: calico/operations/ebpf/enabling-ebpf.mdx [hunk 1]
```diff
@@ -363,16 +363,23 @@ To revert to standard Linux networking:
 
 1. (Depending on whether you installed Calico with the operator or with a manifest) reverse the changes to the operator's `Installation` or the `FelixConfiguration` resource:
 
+<Tabs>
+<TabItem label="Operator" value="Operator-4">
+
```
Comment (ctauchen) [inline calico/operations/ebpf/enabling-ebpf.mdx]: Intent to col 4 to line up with "(Depending ..."

This helps with the layout, so it's clearly part of the step.

File: calico/operations/ebpf/enabling-ebpf.mdx [hunk 2]
```diff
@@ -363,16 +363,23 @@ To revert to standard Linux networking:
...
    ```bash
    kubectl patch installation.operator.tigera.io default --type merge -p '{"spec":{"calicoNetwork":{"linuxDataplane":"Iptables"}}}'
    ```
 
-   or:
+</TabItem>
+<TabItem label="Manifest" value="Manifest-5">
```
Comment (ctauchen) [inline calico/operations/ebpf/enabling-ebpf.mdx]: Indent

File: calico/operations/ebpf/enabling-ebpf.mdx [hunk 3]
```diff
@@ -363,16 +363,23 @@ To revert to standard Linux networking:
...
 
    ```
    calicoctl patch felixconfiguration default --patch='{"spec": {"bpfEnabled": false}}'
    ```
 
+</TabItem>
+</Tabs>
+
```
Comment (ctauchen) [inline calico/operations/ebpf/enabling-ebpf.mdx]: Indent

**Review by ctauchen — COMMENTED** [review]
@StevenTigera Thanks for this! A small formatting fix for you. While you're in there, would you mind making the same changes to OSS 3.24, 3.25, 3.26?

Pending fixes, LGTM

File: calico/operations/ebpf/enabling-ebpf.mdx (hunk 1 above)
Comment (ghost) [reply calico/operations/ebpf/enabling-ebpf.mdx]: Ah OK @ctauchen - I just did simple copy n' paste similar to the existing Tab and TabItem elements for consistency which were flush with the LHS but will Indent and line up as requested

File: calico/operations/ebpf/enabling-ebpf.mdx (hunk 3 above)
Comment (ghost) [reply calico/operations/ebpf/enabling-ebpf.mdx]: Done

File: calico/operations/ebpf/enabling-ebpf.mdx (hunk 2 above)
Comment (ghost) [reply calico/operations/ebpf/enabling-ebpf.mdx]: Done

**Comment by ctauchen** [comment]
Thanks @StevenTigera. That lines up nicely.

Sorry, one more thing from my earlier comment:
> While you're in there, would you mind making the same changes to OSS 3.24, 3.25, 3.26?

**Comment by ghost** [comment]
> @StevenTigera Thanks for this! A small formatting fix for you. While you're in there, would you mind making the same changes to OSS 3.24, 3.25, 3.26?
> 
> Pending fixes, LGTM

@ctauchen - Done - backported changes to OSS 3.24, 3.25, 3.26 as discussed

**Review by bmckercher123 — APPROVED** [review]
LGTM

**Comment by ctauchen** [comment]
> > @StevenTigera Thanks for this! A small formatting fix for you. While you're in there, would you mind making the same changes to OSS 3.24, 3.25, 3.26?
> > Pending fixes, LGTM
> 
> @ctauchen - Done - backported changes to OSS 3.24, 3.25, 3.26 as discussed

@StevenTigera Thanks for the backport. One more fix and we'll be all done.

In your backported code, the code block is out of alignment:
image

If you line those up like your code for Next, you should be all good to go. Thanks!

**Comment by ghost** [comment]
> > > @StevenTigera Thanks for this! A small formatting fix for you. While you're in there, would you mind making the same changes to OSS 3.24, 3.25, 3.26?
> > > Pending fixes, LGTM
> > 
> > 
> > @ctauchen - Done - backported changes to OSS 3.24, 3.25, 3.26 as discussed
> 
> @StevenTigera Thanks for the backport. One more fix and we'll be all done.
> 
> In your backported code, the code block is out of alignment: image
> 
> If you line those up like your code for Next, you should be all good to go. Thanks!

Thanks @ctauchen - IDE re-formatted this before check-in!  Actually, I noticed the original change was 1x sp to the left of "Depending" so I made all 4x files flush with the word "Depending" as requested - this should be all good now

**Comment by ctauchen** [comment]
LGTM

**Pull request merged**